### ISSUE TITLE
Exclude log folder name is wrong

### ISSUE THREAD

**Issue by fsamareanu** [BlackbitDigitalCommerce/pimcore-backup#8]
Hello,

Seems that the name of log folder is defined as var/logs instead of var/log. In my pimcore 10.5 installation the directory var/logs does not exist. The change that needs to be done is this:

```
--- ./pimcore-backup/Command/BackupCommand.php  2022-06-30 09:42:46.000000000 +0400
+++ ./pimcore-backup/Command/BackupCommand.php.new      2022-08-15 13:02:24.704510918 +0400
@@ -94,7 +94,7 @@
         ];

         if(!$input->getOption('only-database')) {
-            $tarFilesCommand = 'tar --exclude=web/var/tmp --exclude=var/tmp --exclude=var/logs --exclude=var/cache --exclude=var/sessions --exclude=var/application-logger'.($input->getOption('skip-versions') ? ' --exclude=var/versions' : '').($input->getOption('skip-assets') ? ' --exclude=var/assets' : '').' --warning=no-file-changed -czf '.$tmpArchiveFilepath.'.gz -C '.PIMCORE_PROJECT_ROOT.' .';
+            $tarFilesCommand = 'tar --exclude=web/var/tmp --exclude=var/tmp --exclude=var/log --exclude=var/cache --exclude=var/sessions --exclude=var/application-logger'.($input->getOption('skip-versions') ? ' --exclude=var/versions' : '').($input->getOption('skip-assets') ? ' --exclude=var/assets' : '').' --warning=no-file-changed -czf '.$tmpArchiveFilepath.'.gz -C '.PIMCORE_PROJECT_ROOT.' .';
             $steps[] = [
                 'description' => 'backup files of entire project root, excluding temporary files',
                 'cmd' => method_exists(Process::class, 'fromShellCommandline') ? Process::fromShellCommandline($tarFilesCommand, null, null, null, null) : new Process($tarFilesCommand, null, null, null, null)
```

**Comment by BlackbitDevs** [comment]
You are right, was `/var/logs` in Pimcore 6. Will adjust it to use the Pimcore constants.

**Comment by BlackbitDevs** [comment]
Hi @fsamareanu, thanks for your report. I have adjusted the bundle to use Pimcore's constants (e.g. `PIMCORE_LOG_DIRECTORY`). Could you give it a try with `dev-master` as version constraint in your composer.json? When it works, I will tag a new release. Thanks in advance.

**Comment by fsamareanu** [comment]
Seems to be working. Thank you.

**Comment by BlackbitDevs** [comment]
Thanks for your feedback, @fsamareanu .
Have released 1.2.8 which includes the fix.
